Makefile

```diff
@@ -4,7 +4,7 @@
 export GO111MODULE := on
 
 # for building out the mycoind app
-BUILDOUT ?= mycoind
+BUILDOUT ?= yourprojectd
```

Comment:
~what does the `d` in the `yourprojectd` stands for?~
I think you can skip the `d` in the name. I think it was supposed to stand for a daemon but I don't think any of those application is one.